scripts/tools/check_includes_config.py

```diff
@@ -157,5 +157,8 @@
 
     # Library meant for non-embedded
     'src/tracing/json/json_tracing.cpp': {'string', 'sstream'},
-    'src/tracing/json/json_tracing.h': {'fstream'}
+    'src/tracing/json/json_tracing.h': {'fstream'},
+
+    # Temporary solution util the OnOff server can provide a callback API for state change.
```

Comment:
Again, please file issue, link to it here.

Reply:
I have raised [this issue #28125](https://github.com/project-chip/connectedhomeip/issues/28125). I will see if I have time to tackle it tomorrow.

Reply:
I meant: put the issue link in the code.

But if you will fix tomorrow, it doesn't matter too much.